blueprints/ember-flexberry-account/index.js

```diff
@@ -0,0 +1,15 @@
+/* globals module */
+module.exports = {
+  afterInstall: function() {
+    var _this = this;
+    return _this.insertIntoFile(
```

Comment:
Also we shoud add `ENV.recaptcha.siteKey` option and `ENV.contentSecurityPolicy` options for recaptcha in `environment.js` on installing addon.